test/common/common/utility_speed_test.cc

```diff
@@ -43,6 +43,23 @@ static void BM_AccessLogDateTimeFormatter(benchmark::State& state) {
 }
 BENCHMARK(BM_AccessLogDateTimeFormatter);
 
+// This benchmark is basically similar with the above BM_AccessLogDateTimeFormatter, the only
+// difference is the format string input for the Envoy::DateFormatter.
+static void BM_DateTimeFormatterWithSubseconds(benchmark::State& state) {
+  int outputBytes = 0;
+
+  static Envoy::SystemTime time(std::chrono::seconds(1522796769));
+  static std::mt19937 prng(1); // PRNG with a fixed seed, for repeatability
+  static std::uniform_int_distribution<long> distribution(-10, 20);
+  static Envoy::DateFormatter date_formatter("%s%3f%5f");
```

Comment:
Why are all these `static`?

Reply:
I admit I did copy and paste from the above `BM_AccessLogDateTimeFormatter`. Will try to modify an see the implication of removing those `static`s.